app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/model/NewsSources.kt

```diff
@@ -17,4 +17,9 @@ data class NewsSources(@PrimaryKey
                        @SerializedName("source")
                        var id: Int = -1,
                        var title: String = "",
-                       var icon: String = "")
\ No newline at end of file
+                       var icon: String = "") {
+
+    val isNewspread: Boolean
+        get() = id == 7
```

Comment:
We have more than one newsspread feed. https://app.tum.de/Api/news/sources

Reply:
Fixed.